ql/src/java/org/apache/hadoop/hive/ql/parse/StorageFormat.java

```diff
@@ -48,9 +51,14 @@ public class StorageFormat {
 
   public enum StorageHandlerTypes {
     DEFAULT(),
-    ICEBERG("\'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler\'",
+    ICEBERG("'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'",
         "org.apache.iceberg.mr.hive.HiveIcebergInputFormat", "org.apache.iceberg.mr.hive.HiveIcebergOutputFormat");
 
+    private static final List<StorageHandlerTypes> SUPPORTED_BY_STORED_BY = Arrays
```

Comment:
nit: Can we rename this to SUPPORTED_STORED_BY?

Reply:
I started feeling we may not need `StorageHandlerTypes.DEFAULT`. But it seems to be a result of a discussion.
https://github.com/apache/hive/pull/4348#discussion_r1239887908

I'm thinking to rename it into like `NON_DEFAULT` for simplicity.

Reply:
Done
https://github.com/apache/hive/pull/4954/commits/136035e9704ea9cd676d0722b83dbd4ee0310039